barpyrus/trayer.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+from barpyrus.widgets import Widget
+from barpyrus.core import EventInput
+from Xlib.display import Display, X
+
+class TrayerWatch(EventInput):
+    def __init__(self, args = { }):
+        command = [ 'trayer' ]
+        self.default_args = {
+            'edge': 'top',
+            'align': 'right',
+            'widthtype': 'request',
+            'expand': 'true',
+            'SetDockType': 'true',
+            'transparent': 'true',
+            'alpha': '0',
+            'height': '16',
+            'margin': '0',
+            'tint': '0x29b2e',
+        }
+        for key,val in args.items():
+            self.default_args[key] = val
+        for key,val in self.default_args.items():
+            command += ["--%s" % (key), str(val)]
+
+        super(TrayerWatch,self).__init__(command)
+        self.proc.stdin.close()
+        self.proc.stdout.close()
+
+        # search for running trayer window
+        self.display = Display()
+        root = self.display.screen().root
+        self.trayer = self.find_tray_window(root)
+        assert (self.trayer != None), 'Panel not found!'
```

Comment:
```suggestion
        assert self.trayer is not None, 'Panel not found!'
```

No need for the parens, and `None` is typically compared with `is`.

Reply:
[x]